lib/tasks/js_rails_routes.rake

```diff
@@ -7,6 +7,7 @@ namespace :js do
     generator.include_names = Regexp.new(ENV['include_names']) if ENV['include_names']
     generator.exclude_names = Regexp.new(ENV['exclude_names']) if ENV['exclude_names']
     generator.output_dir = ENV['output_dir'] if ENV['output_dir']
+    generator.lower_camelcase = ENV['lower_camelcase'] if ENV['lower_camelcase']
```

Comment:
This line becomes `ENV['camlize']`. BTW note that `ENV['lower_camlcase']` is a string or `nil` not boolean.